src/components/Avatar/Debug.stories.tsx

```diff
@@ -33,15 +33,15 @@ Debug.args = {
   shadows: true,
   idleRotation: false,
   headMovement: false,
-  ambientLightColor: '#fff5b6',
-  dirLightColor: '#002aff',
-  spotLightColor: '#fff5b6',
-  ambientLightIntensity: 0.25,
-  dirLightIntensity: 5,
-  spotLightIntensity: 1,
-  dirLightPosition: new Vector3(-3, 5, -5),
-  spotLightPosition: new Vector3(12, 10, 7.5),
-  spotLightAngle: 0.314,
+  keyLightIntensity: 1.2,
```

Comment:
dos it make sense to expose the LIGHT_CONFIG and just use this values? I can imagine doing some tuning on that config that we forget to update on the storybook, what do you think?